cfme/infrastructure/host.py

```diff
@@ -327,7 +327,8 @@ def get_datastores(self):
         datastores = set([])
         for page in paginator.pages():
             for title in sel.elements(
-                    "//div[@id='quadicon']/../../../tr/td/a[contains(@href,'storage/show')]"):
+                    "//div[contains(@id, 'quadicon')]/../../../tr/td/"
+                    "a[contains(@href,'storage/show')]"):
```

Comment:
There is a title, it may have changed place, but there is a title, I checked it earlier I'm almost positive.